scripts/verify-contracts.py

```diff
@@ -0,0 +1,74 @@
+from pathlib import Path
+
+import ape
+import click
+import structlog
+from eth_utils import to_checksum_address
+
+import beamer.deploy.artifacts
+import beamer.deploy.config
+from beamer.typing import URL, ChainId
+
+log = structlog.get_logger(__name__)
+
+
+def _get_contract_info(artifacts_dir: Path) -> dict[ChainId, dict[str, dict[str, str]]]:
+    contract_info: dict[ChainId, dict[str, dict[str, str]]] = {}
+    for file_path in artifacts_dir.glob("*.deployment.json"):
+        deployment = beamer.deploy.artifacts.Deployment.from_file(file_path)
+        if file_path.stem == "base.deployment":
+            ecosystem = "ethereum"
+            chain_id = deployment.base.chain_id
+            contracts = deployment.base.contracts
+        else:
+            assert deployment.chain is not None
+            _, ecosystem = file_path.stem.split(".")[0].split("-")
```

Comment:
Please note there may be several `-` characters in the name. Which part do you care about?

Reply:
1-ethereum.json -> ethereum, xxx-polygon-zkevm -> polygon-zkevm

Reply:
i think i should use it a little different in this manner